chart/values.yaml

```diff
@@ -2,9 +2,11 @@ replicaCount: 1
 
 image:
   repository: oamdev/terraform-controller
-  tag: 0.2.8
+  tag: 0.2.14
```

Comment:
Please keep this line as it is. I am writing a script to automatically replacing 0.2.8 to its latest tag.

Reply:
done